SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -192,6 +204,14 @@ fi
 # configure HOME to be the user's local directory
 HOME=/home/$USER source "${BUILDER_PATH}"
 
+
+# Symlink pyspark (and spark) into the environment to avoid having to install it in the environment
+if [ -n "${INSTALL_NXCALS}" ]; then
+    ENV_LIB=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    ln -s $SPARK_HOME/python/pyspark ${ENV_LIB}
+fi
```

Comment:
As I said above, let's just remove this for now and try to make the whole thing work with a uv install nxcals.